File: C++_Solutions/Project_Euler_Problem_002.cpp
#include <iostream>

int main()
{
	int a = 0, b = 1, c = 0, d = 0;
	while (c < 3500000)
	{
		c = a + b; // Calculates fibonacci sequence starting at 0,1,1,..
		std::cout << c << std::endl;
		a = b; 
		b = c;
		if (c % 2 == 0)
		{d += c;}
	}
	std::cout << "The sum of all EVEN fibonacci values under 4million is: " << d << std::endl;
	return 0;
}

File: Python_Solutions/Project_Euler_Problem_002.py
def main():
    a,b,c,d = 0,1,0,0 
    while (c < 3500000): 
        c = a + b
        a = b
        b = c
        if (c % 2 == 0):
            d += c
    print("The highest fibonacci value under 4million is: " + str(c)) 
    print("The sum of all EVEN fibonacci values under 4million is: " + str(d))
main() 
